java/arcs/core/data/testdata/Request.arcs

```diff
@@ -0,0 +1,29 @@
+// Big example for recipe2plan generation.
```

Comment:
These files all seem way too complicated just for a test. The details of what they are supposed to be testing are completely lost in all the domain-specific names.

Can you please rewrite these to be much clearer in what they are designed to test? e.g. if you just want to test that an import works, maybe call the files something like `Importer.arcs` and `ManifestToImport.arcs` or something like that. And then simplify and rename the particles and schemas to just test the things you need to test, with simpler names.

Reply:
The intention of these manifests is to be a complex, realistic example. They are a broken up version of [this manifest](https://github.com/PolymerLabs/arcs/blob/master/java/arcs/core/data/testdata/IngestProcessRetrieve.arcs).

Reply:
I'd recommend reworking this to be smaller, simpler unit tests instead, that specifically address the test cases you're interested in.

The complex, realistic example will be the actual client usages, which will fail at compile time if there's a problem. We should have a thorough battery of unit tests to check all of the individual edge cases we expect to work, which hopefully means the complex read-world cases don't break.

It's really hard to know what's going on with these tests. If I broke one, I wouldn't know why, or how to fix it. Tests should be simple and readable.

Reply:
I have made these examples simpler. You're right, these should be understandable to the maintainer. We do have unit tests for the rest of the system. These will help users know *why* things are going wrong, in specific. I'll expand on these unit tests to also exercise this feature -- and at a finer granularity.